tailwind.config.js

```diff
@@ -7,8 +7,39 @@ export default {
     "./src/**/*.{js,ts,jsx,tsx}",
   ],
   theme: {
-    extend: {},
-  },
+    colors: {
+      'black': '#1c1c1c',
+      'borders': "#DDDDDD",
+      'bg':'#F3F5F7',
+      'danger-aler-bg': '#FC555619',
+      'unAssigned-bg': "#F7C2D2",
+      'Assigned-bg': "#99BFFB",
+      'InProgress-bg': "#FFE28C",
```

Comment:
Follow a single standard; either separate by hyphen or camelCase; not both;
Also start with lowercase.

Maintain uniformity. Else code will look bad.